WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackSetupWebViewController.swift

```diff
@@ -130,8 +145,58 @@ extension JetpackSetupWebViewController: WKNavigationDelegate {
 private extension JetpackSetupWebViewController {
     enum Constants {
         static let jetpackInstallString = "https://wordpress.com/jetpack/connect?url=%@&mobile_redirect=%@&from=mobile"
-        // TODO: update this URL with woocommerce:// when https://github.com/Automattic/wp-calypso/pull/65715 is merged.
-        static let mobileRedirectURL = "wordpress://jetpack-connection"
+        static let mobileRedirectURL = "woocommerce://jetpack-connected"
+    }
+
+    enum JetpackSetupWebSteps: CaseIterable {
+        case automaticInstall
+        case wpcomLogin
+        case authorize
+        case siteLogin
+        case pluginDetail
+        case pluginInstallation
+        case pluginActivation
+        case pluginSetup
+
+        var path: String {
+            switch self {
+            case .automaticInstall:
+                return "https://wordpress.com/jetpack/connect/install"
+            case .wpcomLogin:
+                return "https://wordpress.com/log-in/jetpack"
+            case .authorize:
+                return "https://wordpress.com/jetpack/connect/authorize"
+            case .siteLogin:
+                return "wp-admin/wp-login.php"
+            case .pluginDetail:
+                return "wp-admin/plugin-install.php"
+            case .pluginInstallation:
+                return "wp-admin/update.php?action=install-plugin"
+            case .pluginActivation:
+                return "wp-admin/plugins.php?action=activate"
+            case .pluginSetup:
+                return "wp-admin/admin.php?page=jetpack"
+            }
+        }
+
+        var trackingStep: WooAnalyticsEvent.LoginJetpackSetupStep {
+            switch self {
+            case .automaticInstall: return .automaticInstall
+            case .wpcomLogin: return .wpcomLogin
+            case .authorize: return .authorize
+            case .siteLogin: return .siteLogin
+            case .pluginDetail: return .pluginDetail
+            case .pluginInstallation: return .pluginInstallation
+            case .pluginActivation: return .pluginActivation
+            case .pluginSetup: return .pluginSetup
+            }
```

Comment:
This looks a bit duplicated, but I want to keep 2 separate enums so that we can reuse the tracking steps for the native flow later.